i18next-scanner.config.js

```diff
@@ -0,0 +1,61 @@
+// This file is part of React-Invenio-Forms
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+const fs = require('fs');
+const chalk = require('chalk');
+
+module.exports = {
+    options: {
+        debug: true,
+        // removeUnusedKeys: true, // removing unused keys
+        browserLanguageDetection: true,
+        func: {
+            list: ['i18next.t', 'i18n.t', 't'],
```

Comment:
What is this list defining? The way you import and mark the strings for translation?

Reply:
yes, list of func name you import and mark the strings for translation?